lib/ansible/modules/cloud/ovirt/ovirt_host_network.py

```diff
@@ -74,7 +74,7 @@
         type: bool
     save:
         description:
-            - "If I(true) network configuration will be persistent, by default they are temporarily."
+            - "If I(false) network configuration will be temporary until next boot, by default they are persistent."
```

Comment:
Can you please write that: Since Ansible 2.8 they are persisted by default? So it's clear.